src/I18n/Number.php

```diff
@@ -276,6 +276,10 @@ public static function formatter($options = [])
             $type = NumberFormatter::CURRENCY;
         }
 
+        if (!empty($options['type']) && $options['type'] === 'ordinal') {
```

Comment:
Wouldn't that make the `type` option a little odd expecting `string` for one thing and `int` for the rest?